helpers/all.js

```diff
@@ -0,0 +1,52 @@
+var _ = require('lodash'),
+    internals = {};
+
+/**
+ * Yield block only if all arguments are valid
+ *
+ * @example
+ * {{#all items theme_settings.optionA theme_settings.optionB}} ... {{/any}}
+ */
+
+internals.implementation = function(handlebars) {
+    this.handlebars = handlebars;
+};
+
+internals.implementation.prototype.register = function(context) {
+    this.handlebars.registerHelper('all', function () {
+
```

Comment:
Sorted! I think I saw in another one of the modules that you're fine with one 'var' declaration so that's the format I've gone with for now. (Is there a public style guide anywhere I've missed for this?)